allen-cell-animated/simularium-website
# Fix/file switching bugs (update viewer to v2.9.5)

Problem
=======
A couple of bugs that have to do with switching trajectories:

[Displayed end time for local file is incorrect when loaded after networked file](https://github.com/allen-cell-animated/simularium-viewer/issues/126)
[Site crashes when switching models](https://github.com/allen-cell-animated/simularium-viewer/issues/127)

Solution
========
Updated the viewer version to 2.9.5, which [fixes the above issues](https://github.com/allen-cell-animated/simularium-viewer/pull/128)

## Type of change
Please delete options that are not relevant.


Steps to Verify:
----------------
1. Pull this branch and `npm i` then `npm start`
2. Verify that the bugs above are no longer reproducible
